Sources/Architecture/ARM/HardwareBreakpointManager.cpp

```diff
@@ -19,6 +19,8 @@ namespace ds2 {
 namespace Architecture {
 namespace ARM {
 
+static const int kMaxHWStoppoints = 32;
```

Comment:
I think this commit is good, but I'm not even sure why we need this logic in the first place. If the CPU says it supports 64 breakpoints, why would we want to limit ourselves to 32?